dark-mode.css

```diff
@@ -719,23 +747,15 @@ body.dark-mode .rcx-css-110cgdy {
 	background-color: transparent !important;
 }
 
-/************** Admin panel & Account panel ******************/
-
-.page-list a:not(.rc-button), .page-settings a:not(.rc-button) {
-	color: var(--primary-font-color);
-}
+/************** All panels ******************/
 
 /*body.dark-mode .simplebar-content > .rcx-box => will also modify sidebar background */
 body.dark-mode .simplebar-content > .rcx-css-fr02gd { /* main content */
     background-color: var(--color-dark);
 }
 
-body.dark-mode .rcx-css-txktj6 { /* Account settings page background */
-    background-color: var(--color-dark) !important;
-}
-
-body.dark-mode .rcx-css-1wm5na { /* Account settings page header title */
-	color: var(--primary-font-color) !important;
+.page-list a:not(.rc-button), .page-settings a:not(.rc-button) {
```

Comment:
Should this have `body.dark-mode` in front of it, or is it meant to always apply?